isis/tests/FunctionalTestsCam2Cam.cpp

```diff
@@ -0,0 +1,53 @@
+#include <iostream>
+#include <QTemporaryFile>
+
+#include "cam2cam.h"
+
+#include "Cube.h"
+#include "CubeAttribute.h"
+#include "IException.h"
+#include "PixelType.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "PvlKeyword.h"
+#include "TestUtilities.h"
+#include "FileName.h"
+#include "ProjectionFactory.h"
+#include "Fixtures.h"
+#include "Mocks.h"
+
+using namespace Isis;
+using ::testing::Return;
+using ::testing::AtLeast;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/cam2cam.xml").expanded();
+
+TEST_F(DefaultCube, FunctionalTestCam2CamNoChange) {
+
+  QVector<QString> args = {"to="+tempDir.path()+"/Cam2CamNoChange.cub", "INTERP=BILINEAR"};
+  UserInterface ui(APP_XML, args);
```

Comment:
I would assume case02 isn't testing anything unique? (all I see is a different input cube.)